src/style.scss

```diff
@@ -3,30 +3,178 @@
+@import './node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariables';
+@import './node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SassCustomProperties';
 
-@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/breakpoints";
-@import "./node_modules/@alaskaairux/orion-web-core-style-sheets/dist/core";
+@import './node_modules/@alaskaairux/orion-web-core-style-sheets/dist/breakpoints';
+@import './node_modules/@alaskaairux/orion-web-core-style-sheets/dist/core';
+@import './node_modules/@alaskaairux/orion-web-core-style-sheets/dist/animation';
 
-// container styles - defines the look of the outer custom componet
-// :host {
-//   ...
-// }
+:host {
+  display: inline-block;
+}
 
-// layout styles - define any layout specifications for UI that is contained WITHIN the component
-// never define layout that would cause effect on element ouside the scope of this component
+:host(.focus-visible) {
+  .pane {
+    border-color: var(--auro-color-border-active-on-light);
+  }
 
+  .pane--selected {
+    // hardcode the token to prevent ShadyCSS from breaking IE11
+    // see https://auro.alaskaair.com/support/shadycss
+    box-shadow: inset 0 0 0 3px #fff;
+  }
+}
 
-// component shape styles
-// .auro-pane {
-//   ...
-// }
+.pane {
+  @include auro_transition(all, 0.15s, ease);
 
-.testClass {
-  color:  var(--auro-color-border-error-on-light);
-  border: 1px solid var(--auro-color-border-error-on-light);
-  display: inline-block;
-  padding: var(--auro-text-body-size-default);
+  // fallback for browsers that don't support grid
+  // also Safari, which does not support display: grid on buttons
+  display: flex; 
+  flex-direction: column;
+  align-items: center;
+
+  @supports (display: grid) {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: 1.625rem 1.25rem;
+  }
+  
+  padding-top: 1.375rem;
+  height: 5.625rem;
+  min-width: 4.25rem;
+
+  font-family: var(--auro-font-family-default);
+  font-weight: var(--auro-text-body-default-weight);
+  background: var(--auro-color-base-gray-100);
+  border: 2px solid;
+  border-color: var(--auro-color-base-gray-200);
+  border-radius: 2.5rem;
+  cursor: pointer;
+
+  @include auro_breakpoint--sm {
+    font-size: var(--auro-text-body-size-default);
+    grid-template-rows: 1.75rem 1.5rem;
+    padding-top: 1.625rem;
+    height: 6.5rem;
+    min-width: 5rem;
+  }
+
+  @media (hover: hover) {
+    &:hover {
+      background: var(--auro-color-border-active-on-light);
+      border-color: var(--auro-color-border-active-on-light);
+
+      .child {
+        color: var(--auro-color-base-white);
+      }
+    }
+  }
+
+  &:active {
+    background: var(--auro-color-ui-active-on-light);
+    border-color: var(--auro-color-ui-active-on-light);
+    transform: scale(0.95);
+  }
+
+  &:disabled {
+    background: none;
+    border-color: transparent;
+    cursor: not-allowed;
+    transform: unset;
+
+    .child {
```

Comment:
This is to disable pointer events on children when the button is disabled.

Would you prefer the following at the root level?
```
.pane--disabled .pane-child {
  pointer-events: none;
}
```

Or is there a better way to write this that I'm missing?

Reply:
When you have a selector that is used independently and doesn't work because the parent/child relationship is not correct, that causes confusion. Being explicit of use regardless of a parent is the desired intent.

Reply:
I think that the missing thing here is that `pane-child` doesn't really explain what this selector will do. It explains moreso placement. 

it feels more like a utility class when I think about it. `.nodeIsDisabled` or something like that is more descriptive as to the intent and responsibility of the selector.

Reply:
I'm at a loss as to how to simplify the child dependency for all parent state's. Disabled and selected are doable since those are driven by props, but I'm not sure how to do :hover and :active (see comment below). This is the only change I've yet to resolve. @blackfalcon, do you have any ideas on how to resolve this for all states?

Reply:
Dale will follow up later. For now it's okay to merge.